build.gradle.kts

```diff
@@ -63,21 +65,30 @@ tasks.withType<Test> {
 detekt {
     config.setFrom("$projectDir/config/detekt.yml")
     autoCorrect = true
+    source.setFrom(
+        files(
+            "$projectDir/src/main/kotlin",
+            "$projectDir/src/test/kotlin",
+            "$projectDir/src/testFixtures/kotlin",
+            "$projectDir/build.gradle.kts",
+        ),
+    )
 }
 
 configure<io.github.surpsg.deltacoverage.gradle.DeltaCoverageConfiguration> {
     diffSource {
         git.compareWith("refs/remotes/origin/master")
     }
-    coverageBinaryFiles = allprojects.asSequence()
-        .map { subproject ->
-            subproject.fileTree(subproject.layout.buildDirectory) {
-                setIncludes(listOf("*/**/*.exec"))
+    coverageBinaryFiles =
+        allprojects.asSequence()
+            .map { subproject ->
+                subproject.fileTree(subproject.layout.buildDirectory) {
+                    setIncludes(listOf("*/**/*.exec"))
+                }
+            }
+            .fold(files()) { all, files ->
+                all.from(files)
             }
-        }
-        .fold(files()) { all, files ->
-            all.from(files)
-        }
     violationRules.failIfCoverageLessThan(0.9)
     reports {
         html.set(true)
```

Comment:
In the 2.3.0 version the markdown report is available, it looks like this
![image](https://github.com/user-attachments/assets/2f6b8a01-9a26-4904-a363-6ec1e234a283)

Reply:
we use 2.3.0
we can make ticket and increace version in another pr